lib/ansible/modules/windows/win_get_url.ps1

```diff
@@ -236,22 +467,48 @@ if ([Net.SecurityProtocolType].GetMember("Tls12").Count -gt 0) {
 }
 [Net.ServicePointManager]::SecurityProtocol = $security_protocols
 
+# Check for case $checksum variable contain url. If yes, get file data from url and replace original value in $checksum
+if ($checksum_url) {
+
+    if ($checksum_url.startswith('http://', 1) -or $checksum_url.startswith('https://', 1) -or `
```

Comment:
I prefer be explicit and actually put the enum name for [StringComparison](https://docs.microsoft.com/en-us/dotnet/api/system.stringcomparison?view=netframework-4.7.2). I would also use `InvariantCultureIgnoreCase` instead as well.

Reply:
Fixed